master/buildbot/schedulers/base.py

```diff
@@ -151,6 +178,13 @@ def _changeCallback(self, key, msg, fileIsImportant, change_filter,
                     'not processed by scheduler %(name)s',
                     codebase=change.codebase, name=self.name)
             return
+
+        if not self.enabled:
```

Comment:
I'm not sure I understand. I'm following the same logic to ignore changes as the isFileImportant callback. So it will ignore the changes when the scheduler isn't on. Are you suggesting it SHOULD remember? If so then should I disable the change consumer on the enable toggle?